images/70-qemu/archs

```diff
@@ -0,0 +1 @@
+amd64
```

Comment:
Missing `arm64` and it looks like it is supported for `libguestfs-tools` on `debian:stable`.